alluxiofs/client/worker_ring.py

```diff
@@ -150,31 +148,29 @@ def from_host_and_port(worker_host, worker_http_port):
 
 
 class EtcdClient:
-    def __init__(self, host="localhost", port=2379, options=None):
+    def __init__(
+        self, config: AlluxioClientConfig, host="localhost", port=2379
```

Comment:
could we not use default host and port here, but the etcd_hosts and port in the configuration

Reply:
Please take a look the code logics, i forget how we deal with multiple etcd hosts